package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Skynet REST API",
   "main": "index.js",
   "scripts": {
-    "test": "npx jest --passWithNoTests",
+    "test": "jest --passWithNoTests --watch",
```

Comment:
@PedroLabrador this --watch will keep the build phase for ever, we might want to create a separate command for the pipelines let me change it

Reply:
ok sure